targets/frontend/src/components/layout/Nav.js

```diff
@@ -115,9 +121,20 @@ export function Nav() {
   );
 }
 
+const editableSources = [SOURCES.EDITORIAL_CONTENT];
+
 function ActiveLink({ children, href }) {
   const router = useRouter();
-  const isCurrentRoute = router.asPath.startsWith(href);
+  const [pathname, query = ""] = href.split("?");
+  let isCurrentRoute = router.pathname === pathname;
+  if (isCurrentRoute) {
+    if (query) {
+      isCurrentRoute = query.includes(router.query?.source);
+    } else {
+      isCurrentRoute = !editableSources.includes(router.query?.source);
```

Comment:
// we don't want the link `contenus` to be highlighted if another link is already highlighted

Reply:
should this work with `["editorial_content"].includes(router.query?.source)`

Reply:
I'm not sure I get what you mean ?

Reply:
I wonder if we could transform the negation on route we "mark" as subroute (`/contenus?source=editorial_content`)

Reply:
on regarde ça demain matin durant le pair programming si tu veux parce que là j'arrve pas à faire connecter ce que tu suggères dans ma tête